liquidity-rewards/contracts/UniswapPoolRewards.sol

```diff
@@ -44,7 +44,7 @@ contract UniswapPoolRewards is PoolTokenWrapper, Ownable {
     /**
      * Deposit the reward token and update the reward.
      */
-    function depositReward(uint256 reward) external onlyOwner {
+    function depositReward(uint256 reward) external {
```

Comment:
@hhio618 What if someone makes a regular ERC-20 transfer like this one?: https://rinkeby.etherscan.io/tx/0x426aacd862020c6d6ceafaa5d9b5b6e1b6e81a0f6bf1fcef6437278b41bcc3b1

In that case, will `_updateReward();` still be called?